cluster/stone-prd-rh01/tenants/rh-acs-tenant/integration_test_scenario.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  name: acs-enterprise-contract-allow-quay-registry
+  namespace: rh-acs-tenant
+spec:
+  application: acs
+  contexts:
+    - description: Application testing
+      name: application
+  resolverRef:
+    params:
+      - name: url
+        value: 'https://github.com/redhat-appstudio/build-definitions'
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: pipelines/enterprise-contract-redhat-no-hermetic.yaml
+    resolver: git
+  params:
+  - name: POLICY_CONFIGURATION
+    value: github.com/stackrox/stackrox?ref=rhtap-allow-quay-registry//.tekton/enterprise-contract
```

Comment:
Usually what we are doing with the policies is create them in the same repo as the ReleasePlanAdmission and reference them here using the `namespace/name` format of referencing resources in Kubernetes. I'm not sure how this works so I will let @scoheb check it out as well

Reply:
I found an example in https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/blob/main/config/OP/EnterpriseContractPolicy/registry-rhtas.yaml?ref_type=heads and will adapt it. I assume the changes there need to be merged and provisioned before this PR can be merged?

Reply:
Proposal for the policy: https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/merge_requests/184 .

Reply:
@tommartensen https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/merge_requests/184 looks good. But I cannot approve. 

@ernesgonzalez33 you are bang on!

Reply:
@ernesgonzalez33 @scoheb 

The MR was approved and merged. I updated the policy configuration to namespace/name format. 
Can you confirm that it exists and approve this PR? 
Thanks.